src/Appwrite/Platform/Workers/Migrations.php

```diff
@@ -222,8 +237,23 @@ protected function processMigration(Document $migration, Realtime $queueForRealt
         $projectDocument = $this->dbForPlatform->getDocument('projects', $project->getId());
         $tempAPIKey = $this->generateAPIKey($projectDocument);
 
+        $importDocument = null;
         $transfer = $source = $destination = null;
 
+        if ($migration->getAttribute('source') === Csv::getName()) {
+            $fileSize = $migration->getAttribute('credentials', [])['size'] ?? 0;
+            $importDocument = new Document([
+                '$id' => ID::unique(),
+                'size' => $fileSize, // uncompressed and decrypted file size
+                'startedAt' => DateTime::now(),
+                'migrationId' => $migration->getId(),
+                'migrationInternalId' => $migration->getInternalId(),
+                'resourceId' => $migration->getAttribute('resourceId', ''),
+                'resourceType' => $migration->getAttribute('resourceType', ''),
+                'errors' => [],
+            ]);
```

Comment:
Maybe we don't need this collection at all :thinking: if we have `options` on migrations, we can add size/path etc there, and we can differentiate what will be in options based on source/destination combo

Reply:
Makes sense to me. But then its an `option` attribute set only for `csv` led migration [atleast atm]. wdyt?